Source/NSCollectionView.m

```diff
@@ -525,12 +807,12 @@ - (void) tile
   NSSize itemSize = NSMakeSize(_minItemSize.width, _minItemSize.height);
   
   _numberOfColumns = MAX(1.0, floor(width / itemSize.width));
-
+  
```

Comment:
There seem to be a few lines with trailing spaces.

Reply:
This has been resolved.  I have eliminated as much whitespace as I could find.  I am tempted to write a script to do this automatically when needed.

Reply:
Most other trailing whitespace are gone, but this specific is still there. Doesn't your editor have a command for removing these?

Reply:
I'm using emacs.  I just now ran cleanup-whitespace (I didn't know it had that feature) so, hopefully, that resolves everything.

Reply:
@fredkiefer Please let me know.  I just pushed.

Reply:
I am resolving this as I believe all of the whitespace should be gone.